### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by srcnalt**
Folder structure and some classes are refactored.

### CODE REVIEW

File: Editor/UI/EditorWindows/EditorWindowBase.cs [hunk 1]
```diff
@@ -54,34 +59,33 @@ private void LoadAssets()
...
-            if (webButtonStyle == null)
+            webButtonStyle ??= new GUIStyle(GUI.skin.button)
             {
-                webButtonStyle = new GUIStyle(GUI.skin.button);
-                webButtonStyle.fontSize = 12;
-                webButtonStyle.fixedWidth = 149;
-                webButtonStyle.fixedHeight = ButtonHeight;
-                webButtonStyle.padding = new RectOffset(5, 5, 5, 5);
-            }
+                fontSize = 12,
```
Comment (HarrisonHough) [inline Editor/UI/EditorWindows/EditorWindowBase.cs:76]: minor thing but we have font size hardcoded to 12 in two places, we could instead store it in a constant.

File: Editor/UI/EditorWindows/EditorWindowBase.cs [hunk 2]
```diff
@@ -12,35 +12,40 @@ namespace ReadyPlayerMe.Core.Editor
...
-        /// 
-        /// </summary>
         private void LoadAssets()
         {
-            banner ??= new Banner();
+            banner ??= new Banner(editorWindowName);
+            
+            footer ??= new Footer(editorWindowName);
+            
+            // TODO: move this into avatar loader settings window
```
Comment (HarrisonHough) [inline Editor/UI/EditorWindows/EditorWindowBase.cs:38]: para-phrasing a quote from Pablo "TODO's never get done" :D probably we could remove

**Review by HarrisonHough — APPROVED** [review]
some minor things in comments but all good otherwise

File: Editor/UI/EditorWindows/EditorWindowBase.cs (hunk 2 above)
Comment (srcnalt) [reply Editor/UI/EditorWindows/EditorWindowBase.cs:38]: shit forgot this here, will take a look now

**Pull request merged**